src/components/try-it/copy-button/CopyButton.tsx

```diff
@@ -0,0 +1,13 @@
+"use client";
+
+export default function CopyButton() {
+    function copyRequestURL(event) {
```

Comment:
Either remove event (as it's not needed after changing this function to how I said in another comment), or put the type in here.